perf/src/main/scala/cromwell/perf/CompareMetadata.scala

```diff
@@ -0,0 +1,111 @@
+package cromwell.perf
+
+import java.time.Duration
+
+import better.files.File
+import cats.data.Validated.{Invalid, Valid}
+import cats.implicits._
+import common.validation.ErrorOr._
+import io.circe
+import io.circe.generic.auto._
+import io.circe.parser._
+
+// Do not remove this unused import
+// If removed, circe parser fails to find implicit decoder for OffsetDateTime, and parsing fails
+
+object CompareMetadata extends App {
+
+  def parseMetadataFromFile(filePath: String): Either[circe.Error, Metadata] = {
+    val metadataFile = File(filePath)
+    val metadataFileContent = metadataFile.contentAsString
+
+    decode[Metadata](metadataFileContent)
+  }
+
+
+  def displayComputedMetrics(metadata: Metadata, displayMsg: String): Unit = {
+    println(displayMsg)
+
+    println(s"Workflow started after: ${metadata.workflowStartedAfter}")
+    println(s"Workflow Running time: ${metadata.workflowRunningTime}")
+    println(s"Total jobs per root workflow: ${metadata.totalJobsPerRootWf}")
+    println(s"Avg cache copy retries: ${metadata.avgCacheRetries}")
+    println(s"Avg time job spent in Call Caching state: ${metadata.avgTimeInCallCachingState}")
+    println(s"Avg time job spent in Job Preparation state: ${metadata.avgTimeInJobPreparation}")
+    println(s"Avg time job spent in fetching and copying cache hit(s) state: ${metadata.avgTimeForFetchingAndCopyingCacheHit}")
+  }
+
+  def compareDurationMetrics(metadataOld: Metadata, metadataNew: Metadata, metricFunc: Metadata => Duration, metricName: String): ErrorOr[String] = {
+    if(metricFunc(metadataNew).toMillis > 1.1 * metricFunc(metadataOld).toMillis){
```

Comment:
can we pull this 1.1 out to a constant in case we ever decide the magic difference is not 10%?

Reply:
Good point